README.md

```diff
@@ -260,3 +264,60 @@ Redis Operator Sample `values.yaml` -
 
 - View an example of how to use the Redis Operator to deploy a Redis standalone instance [here](supervisor-services-labs/redis-operator/redis-instance.yaml)
 - For advanced configurations, check the [detailed reference](https://ot-redis-operator.netlify.app/docs/getting-started/).
+
+## RabbitMQ Cluster Kubernetes Operator
```

Comment:
Not sure why this MR has the RabbitMQ changes?

Reply:
I think there was a typo that I fixed in the RabbitMQ section.

Reply:
Fixing this.